tests/integrations/tests/test_openai.py

```
@@ -1580,16 +1605,978 @@ def test_40_text_completion_streaming(self, openai_client, test_config):
+                print(f"  Verified batch {batch.id} is in list")
+
+                print(
+                    f"Success: Inline batch E2E completed for batch {batch.id} (provider: {provider})"
+                )
+
+            finally:
+                if batch:
+                    try:
+                        client.batches.cancel(batch.id, extra_body={"provider": provider})
+                        print(f"Cleanup: Cancelled batch {batch.id}")
+                    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)